ui_testing/testcases/basic_tests/test03_orders.py

```diff
@@ -0,0 +1,56 @@
+from ui_testing.testcases.base_test import BaseTest
+from ui_testing.pages.login_page import Login
+from ui_testing.pages.article_page import Article
+from ui_testing.pages.testplan_page import TestPlan
+from ui_testing.pages.order_page import Order
+from ui_testing.pages.analyses_page import Analyses
+
+class OrdersTestCases(BaseTest):
+    def setUp(self):
+        super().setUp()
+        self.login_page = Login()
+        self.article_page = Article()
+        self.test_plan = TestPlan()
+        self.order_page = Order()
+        self.analyses_page = Analyses()
+        self.login_page.login(
+            username=self.base_selenium.username, password=self.base_selenium.password)
+        self.base_selenium.wait_until_page_url_has(text='dashboard')
+        self.order_page.get_orders_page()
+
+    def test01_archiveOrder(self):
+        """
+            New: Orders: Archive
+            The user cannot archive an order unless all corresponding analysis are archived
+            LIMS-3425
+        :return:
+        """
+        order_row = self.order_page.get_random_order_row()
+        self.order_page.click_check_box(source=order_row)
+        order_row_array = order_row.text.split('\n')
+
+        headers = self.order_page.table_headers()
+        headers = list(filter(lambda x: (x != ''), [
+                       header.text for header in headers]))
+        analysis_index = headers.index('Analysis No.')
+        analysis_numbers_array = order_row_array[analysis_index].split(',')
```

Comment:
from line 30 to line 36 should be a generic method which takes any_row_list and the filter_by paramere, then it returns the value of the table cell